specification/agrifood/data-plane/Microsoft.AgFoodPlatform/preview/2022-11-01-preview/examples/Scenes_SearchFeatures.json

```diff
@@ -433,8 +433,7 @@
             }
           }
         ],
-        "next": 17,
-        "returned": 2
+        "nextLink": "https://{{resourceName}}.farmbeats.azure.net//scenes/stac-collections/{collectionId}:search?api-version={{apiVersion}}&collectionId=Sentinel_2_L2A&maxpagesize=2&skip=17"
```

Comment:
`collectionId=Sentinel_2_L2A` part seems wrong?

`collectionId` should not be in query parameter, but in path e.g. `https://{{resourceName}}.farmbeats.azure.net//scenes/stac-collections/Sentinel_2_L2A:search?...`

I know this is an example, but make sure your backend is correct.

Reply:
It was manual error in example. Updated it. Thanks!